docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+| PaginationRequestBuilderPlugin                        | Extends `GlueRequestTransfer.pagination` with `PaginationTransfer`.             | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| SortRequestBuilderPlugin                              | Extends `GlueRequestTransfer.sorting` with `SortTransfer`.                      | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| FilterFieldRequestBuilderPlugin                       | Extends `GlueRequestTransfer.filter` with `GlueFilterTransfer`.                 | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| RelationshipResponseFormatterPlugin                   | Loads resource relationships dependencies and adds included relationships.      | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| JsonApiResponseFormatterPlugin                        | Builds response for the JSON API convention.                                    | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| LocaleRequestBuilderPlugin                            | Extends `GlueRequestTransfer.locale` with `GlueRequestTransfer.meta`.           | Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication                               |
+| SecurityHeaderResponseFormatterPlugin                 | Extends `GlueResponseTransfer` with security headers.                           | Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication                               |
+| CustomRouteRouterPlugin                               | Routes API request based on `RouteProviderPluginInterface`.                     | Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication                               |
+| CustomRouteRouteMatcherPlugin                         | Routes the API request using the custom routes.                                 | Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication                               |
+| RequestCorsValidatorPlugin                            | Validates cors headers.                                                         | Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication                               |
+| StorefrontApiRelationshipProviderPlugin               | Provides relationships provider plugins.                                        | Spryker\Glue\GlueStorefrontApiApplicationGlueJsonApiConventionConnector\Plugin\GlueStorefrontApiApplication |
```

Comment:
Provides a collection of resource relationships for the storefront API application.